020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,194 @@
+| `CALL`                  | N             | N              | Y                 |
+| `ALTER STATISTICS`      | N             | N              | Y                 |
+| `CREATE STATISTICS`     | N             | N              | Y                 |
+| `DROP STATISTICS`       | N             | N              | Y                 |
+| `ALTER RULE`            | N             | N              | Y                 |
+| `CREATE RULE`           | N             | N              | Y                 |
+| `DROP RULE`             | N             | N              | Y                 |
+| `ALTER TYPE`            | N             | N              | Y                 |
+| `CREATE TYPE`           | N             | N              | Y                 |
+| `DROP TYPE`             | N             | N              | Y                 |
+| `ALTER DOMAIN`          | N             | N              | Y                 |
+| `CREATE DOMAIN`         | N             | N              | Y                 |
+| `DROP DOMAIN`           | N             | N              | Y                 |
+| `ALTER FUNCTION`        | N             | N              | Y                 |
+| `CREATE FUNCTION`       | N             | N              | Y                 |
+| `DROP FUNCTION`         | N             | N              | Y                 |
+| `EXPLAIN`               | N             | N              | Y                 |
+| `REINDEX`               | N             | N              | Y                 |
+| `ALTER OPERATOR CLASS`  | N             | N              | Y                 |
+| `CREATE OPERATOR CLASS` | N             | N              | Y                 |
+| `DROP OPERATOR CLASS`   | N             | N              | Y                 |
+| `CREATE EXTENSION`      | N             | N              | Y                 |
+| `DROP EXTENSION`        | N             | N              | Y                 |
+| `ALTER EXTENSION`       | N             | N              | Y                 |
+| Temporary tables        | N             | N              | Y                 |
+
+## Limitations
+
+### Unsupported statements and functions
+
+The following statements and/or functions are not available:
+
+- `SELECT user;`
+- `SHOW password_encryption;`
+
+### Connection limits
+
+- SQL over HTTP to a shared cluster has the rate limits in place, for a dedicated cluster there are none.
+- Shared cluster - TODO, amount of available connections
+- Dedicated cluster - no limit to connections
+
+### `xata_` Column prefix
+
+Xata reserves the `xata_` column prefix for [special columns](docs/concepts/data-model#special-columns) and rejects any custom created columns with the prefix.
+
+### sslmode
+
+Xata does not allow plain text connections, this may have an effect on the client you are using and the default that is inferred by such.
+If you face an issue, be append the SSL mode `require` option to the connection string or pass in a parameterized way depending on the tool you are using.
```

Comment:
I would skip the appending `require` suggestion since in `psql` the default behaviour is attempting SSL, so this wouldn't make any difference if the user is facing a connectivity issue.

However, I would provide some guidelines about which mode to use in `psql`, something like:

Xata recommends that the client verifies the server certificate, in `psql` this would be achieved with the `verify-ca` and `verify-full` modes. We will need to provide a certificate store to do this, it is usually present in most operative systems, for instance:
- MacOS: /etc/ssl/cert.pem
- GNU/Linux: /etc/ssl/certs/ca-certificates.crt

Using `psql` on Mac to establish a fully secure connection to Xata is as easy as:

```
psql "sslmode=verify-full sslrootcert=/etc/ssl/cert.pem user= ws1234 password=xau_apikey123456 host=us-east-1.sql.xata.sh dbname=Games:main"
```

Keep in mind that different clients will provide different configuration options.

Reply:
Thank you for the extended feedback Alejandro, I'll address and ping you.

Reply:
first draft via https://github.com/xataio/mdx-docs/pull/281/commits/5e0b38b19a8f156e0768e82e6e19308a7393523a